src/Components/Header/Header.js

```diff
@@ -35,7 +38,7 @@ const Header = () => {
             >
               <span className="material-icons"> favorite </span>
               <span className="badge badge-round secondary-bg-color dark-text">
-                5
+                {authState.isLoggedIn ? wishList.length : 0}
```

Comment:
Suggestion: No badge should be displayed on the Wishlist icon if the user is not logged in (not even 0)

Reply:
Ok. I will change this